plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchServer.java

```diff
@@ -50,7 +51,7 @@
 import static java.nio.file.Files.createTempDirectory;
 import static org.testcontainers.utility.MountableFile.forHostPath;
 
-public class ElasticsearchServer
+public class ElasticsearchServer implements Closeable
```

Comment:
Hi @striderarun, In first commit `Make ElasticsearchServer Closeable` you have to correct the style.

```suggestion
public class ElasticsearchServer
        implements Closeable
```

Reply:
Thanks so much for the pointer @krvikash. My confusion was because though the style of ElasticsearchServer.java was incorrect in the first commit, I had later corrected it in the fourth commit of the PR. I incorrectly thought the CI would look at only the final version of ElasticsearchServer.java when checking the style.

Your comment helped me realize that's not the case and every commit has to adhere to style guideline.

Reply:
Also, I have already submitted the signed CLA to cla@trino.io, but looks like it's not yet been registered.